### PR TITLE
MyFT notifications feedback.

### PR DESCRIPTION
**Pull request by dan-searle**
🐿 v2.8.0

### CODE REVIEW

File: myft-notification/notification-expander.html [hunk 1]
```diff
@@ -13,4 +13,14 @@
 			<div class="myft-notification__link--all-stories">See all {{items.length}} stories</div>
 		</a>
 	</div>
+
+	<div class="myft-notification__feedback" data-feedback-feature="myft-notification">
+		<p class="myft-notification__feedback-question">Are these notifications useful?</p>
+		<button class="myft-notification__feedback-button js-feedback__responder" data-answer="positive" data-question="myft-notifications-helpful">
+			Yes it's useful
```
Comment (benbarnett) [inline myft-notification/notification-expander.html]: [pedant klaxon] could we use a curly quote for the apostrophe? `’`

**Comment by benbarnett** [comment]
Neat 👍

**Comment by dan-searle** [comment]
More to do...

**Comment by dan-searle** [comment]
Yeah, it seems better in `components/`. I'll move it.

**Pull request merged**